### PR TITLE
Update license file (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,14 +1,29 @@
-Copyright (c) 2005-2016, Regents of the University of California, the University of North Carolina at Chapel Hill, and the Data Intensive Cyberinfrastructure Foundation
-All rights reserved.
+BSD 3-Clause License
 
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+Copyright (c) 2005-2024, The University of North Carolina at Chapel Hill
```
Comment (trel) [inline LICENSE:3]: i don't think we can drop the other two copyright holder notices.   we *could* put them on separate lines, noting their increasing age...

Comment (korydraughn) [reply LICENSE:3]: Oh really? I had no idea.

Should the year(s) be identical?

Comment (korydraughn) [reply LICENSE]: Updated.

**Review by alanking — COMMENTED** [review]
Deferring :)

File: LICENSE [hunk 2]
```diff
@@ -1,14 +1,30 @@
-Copyright (c) 2005-2016, Regents of the University of California, the University of North Carolina at Chapel Hill, and the Data Intensive Cyberinfrastructure Foundation
-All rights reserved.
+BSD 3-Clause License
 
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+Copyright (c) 2005, Regents of the University of California
+Copyright (c) 2008, The University of North Carolina at Chapel Hill
```
Comment (trel) [inline LICENSE:4]: let's flip these so unc on top - then i think we're good

Comment (korydraughn) [reply LICENSE]: Done.

**Review by trel — APPROVED** [review]

**Comment by korydraughn** [comment]
#'d

**Comment by trel** [comment]
and for the record.... no material changes - just cosmetic / consistency with other repositories under the iRODS organization.

**Pull request merged**